openapi/paths/risk-score-simulations.yaml

```
@@ -0,0 +1,65 @@
+get:
+  x-badge: Experimental
+  x-products:
+    - Core
+  tags:
+    - Risk score
+  summary: Retrieve a risk score simulations
+  operationId: GetRiskScoreSimulationsCollection
+  x-sdk-operation-name: getAll
+  security:
+    - SecretApiKey: []
+    - JWT: []
+  description: Retrieves a risk score simulations.
```

nitpick: (I think this is gets all)
```suggestion
  description: Retrieves risk score simulations.
```